rtsp.go

```diff
@@ -1044,6 +1178,43 @@ func NewRTSPCamera(ctx context.Context, _ resource.Dependencies, conf resource.C
 	return rc, nil
 }
 
+// getHomeDir returns the home directory of the user.
+func getHomeDir() (string, error) {
+	home, err := os.UserHomeDir()
+	if err != nil {
+		return "", err
+	}
+	return home, nil
+}
+
+func applyStorageDefaults(c videoStoreStorageConfig, name string) (videostore.StorageConfig, error) {
```

Comment:
Discussed in person moving default and getHomeDir helpers back into video-store repo.